### PR TITLE
adds gcrypt to systemd-journeld

### PR DESCRIPTION
**Pull request by x85446**

### CODE REVIEW

**Comment by psppalash** [comment]
Will test the build on raspberry Pi. We can merge the PR after that.

File: recipes-core/systemd/systemd-conf%.bbappend [hunk 1]
```diff
@@ -0,0 +1,6 @@
+SUMMARY = "adds flags to journeld.conf in a simlar way to the metod used by systemd-conf found in the systemd bitbake reciepe"
```
Comment (cmonr) [inline recipes-core/systemd/systemd-conf%.bbappend:1]:
```suggestion
SUMMARY = "adds flags to journeld.conf in a simmilar way to the method used by systemd-conf found in the systemd bitbake reciepe"
```

File: recipes-core/systemd/systemd_%.bbappend [hunk 2]
```diff
@@ -0,0 +1,2 @@
+SUMMARY = "adds gcrypt (libgcrypt) to systemd binary for forward secure sealing (ffs) journeld"
```
Comment (cmonr) [inline recipes-core/systemd/systemd_%.bbappend:1]:
```suggestion
SUMMARY = "adds gcrypt (libgcrypt) to systemd binary for forward secure sealing (ffs) journald"
```

**Review by cmonr — CHANGES_REQUESTED** [review]
Typos.

**Review by ygoyal18 — APPROVED** [review]

**Comment by psppalash** [comment]
Verified. We can merge this PR now.

**Review by psppalash — APPROVED** [review]

**Review by cmonr — APPROVED** [review]

**Pull request merged**